HW_1/Task_4.py

```diff
@@ -0,0 +1,11 @@
+# Требуется определить, можно ли от шоколадки размером n
+# × m долек отломить k долек, если разрешается сделать один разлом по
+# прямой между дольками (то есть разломить шоколадку на два
+# прямоугольника).
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n,m,k = int(input('В-те 1-ю сторону: ')),int(input('В-те 2-ю сторону: ')),int(input('В-те кол-во долек: '))
+if k%n == 0 or k%m == 0:
```

Comment:
```suggestion
if k % n == 0 or k % m == 0:
```
Это условие обязательное, но недостаточное. Например, если ввести 3, 2 и 6, то ваша программа выведет положительный результат. На самом деле отломить такое количество долек нельзя, потому что разлома, как такового, не будет.